content/coronavirus_landing_page.yml

```diff
@@ -3,14 +3,20 @@ content:
   meta_description: "Find information on coronavirus, including guidance, support, announcements and statistics."
   page_header: "Coronavirus (COVID&#8209;19)"
   stay_at_home:
-    pretext: Stay at home
+    pretext: Stay alert
     list:
-      - Only go outside for food, health reasons or work (but only if you cannot work from home)
-      - If you go out, stay 2 metres (6ft) away from other people at all times
-      - Wash your hands as soon as you get home
+      - Stay at home as much as possible
```

Comment:
Are we planning to make the first letter lowercase @samdub ?

Reply:
I think Sam said in standup that we're letting it sit for a bit as it came from Number 10. 🤷